R/article.R

```diff
@@ -256,6 +256,16 @@ informs_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
   )
 }
 
+#' @section `iop_article`: Format for creating submissions to
+#'   IOP journals. Adapted from \samp{https://publishingsupport.iopscience.iop.org/questions/latex-template/}.
+#' @export
+#' @rdname article
+iop_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
```

Comment:
See my main remark on `citation_package`. Other value can be passed here, but I don't think the template will work in the current state.